src/Common/AzurePSCmdlet.cs

```diff
@@ -434,6 +438,37 @@ protected bool IsVerbose()
             return verbose;
         }
 
+        protected void WriteSurvey()
+        {
+            HostInformationMessage msg = new HostInformationMessage()
+            {
+                Message = string.Format("Survey: How was your experience using {0}.{1}?`\nRun ", ModuleName, ModuleVersion),
```

Comment:
Since we don't raise survey per module. Let's use Azure PowerShell for the time being.